CHANGELOG

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Jul 13 07:40:52 UTC 2016 - jreidinger@suse.com
+
+- Do not crash for submenu with double quotes (bnc#986005)
+- filter out snapper menuentry ( using double quotes )
```

Comment:
why should a generic grub config be interested in snapper? Did you mean to say this instead?
```
- Handle submenus with double quotes (do not crash)
- Filter out menuentries with double quotes (used by snapper in opensuse/SLE)
```